pkg/networkservice/common/mechanisms/vxlan/vni/client.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020-2021 Cisco and/or its affiliates.
```

Comment:
```suggestion
// Copyright (c) 2021 ${TODO: Add your comany name} and/or its affiliates.
```

Reply:
This file is copied from sdk-vpp [repo](https://github.com/networkservicemesh/sdk-vpp/tree/main/pkg/networkservice/mechanisms/vxlan/vni), so retaining the copyright here.

Reply:
OK, then be free to append a copyright holder as in example https://github.com/networkservicemesh/sdk/tree/main/.ci/license#copyright-headers-for-source-code

Reply:
done